Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -69,7 +69,7 @@ The most basic form of a preview is when a live record is selected and
 you lookup a future version of that record belonging to the current
 workspace of the logged in backend user. This is very easy as long as
 a record is selected based on its "uid" or "pid" fields which are not
-subject to versioning; You simply call :code:`sys_page->versionOL()` after
+subject to versioning; You call :code:`sys_page->versionOL()` after
```

Comment:
```suggestion
subject to versioning; Call :code:`sys_page->versionOL()` after
```

Reply:
Nope, here not: https://libguides.royalroads.ca/punctuation-capitalization/capsemicolons

Reply:
The whole semicolons in this part do not belong here. Will have a look.